code/datums/explosion.dm

```diff
@@ -222,6 +222,12 @@ GLOBAL_LIST_EMPTY(explosions)
 				var/throw_range = rand(throw_dist, max_range)
 				var/turf/throw_at = get_ranged_target_turf(I, throw_dir, throw_range)
 				I.throw_at(throw_at, throw_range, EXPLOSION_THROW_SPEED)
+		
+		for(var/mob/living/L in T)
```

Comment:
I kinda wonder if this loop and the above loop could just be merged into one `for(var/atom/A in T)`

Reply:
You're *probably* right, we'll see

Are effects anchored?

Reply:
nah, i can't merge it together. Atoms dont have a base anchor var or a throw_at() proc, so i'd have to call isitem and isliving, which means we just pointlessly loop through all atoms here